labelbox/schema/annotation_import.py

```diff
@@ -454,3 +454,155 @@ def _create_mea_import_from_bytes(
         res = cls._create_from_bytes(client, variables, query_str, file_name,
                                      bytes_data)
         return cls(client, res["createModelAssistedLabelingPredictionImport"])
+
+
+class LabelImport(AnnotationImport):
```

Comment:
I think we might want the ability to use a local file, also, like bulkImportRequest?

Reply:
It looks like you added that?

Reply:
i started to, then i stopped and undid it, figured i would ask first.